cups.plugin fails to be compiled

```
 CC       collectors/proc.plugin/sys_kernel_mm_ksm.o
  CC       collectors/proc.plugin/sys_devices_system_edac_mc.o
  CC       collectors/proc.plugin/sys_devices_system_node.o
  CC       collectors/proc.plugin/sys_fs_btrfs.o
  CC       collectors/proc.plugin/sys_class_power_supply.o
  CC       collectors/tc.plugin/plugin_tc.o
  CCLD     apps.plugin
  CCLD     cgroup-network
  CCLD     freeipmi.plugin
  CCLD     cups.plugin
/usr/bin/ld: libnetdata/log/log.o: in function `fatal_int':
/home/costa/src/netdata-ktsaou.git/libnetdata/log/log.c:410: undefined reference to `send_statistics'
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:1772: cups.plugin] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/home/costa/src/netdata-ktsaou.git'
make[1]: *** [Makefile:2669: all-recursive] Error 1
make[1]: Leaving directory '/home/costa/src/netdata-ktsaou.git'
make: *** [Makefile:1442: all] Error 2
 FAILED   
```

--- a/collectors/cups.plugin/cups_plugin.c
+++ b/collectors/cups.plugin/cups_plugin.c
@@ -14,6 +14,13 @@ void netdata_cleanup_and_exit(int ret) {
     exit(ret);
 }
 
+void send_statistics( const char *action, const char *action_result, const char *action_data) {
+    (void) action;
+    (void) action_result;
+    (void) action_data;
+    return;
+}
+
 // callbacks required by popen()
 void signals_block(void) {};
 void signals_unblock(void) {};

